source/includes/mongodb-compatibility-table-rust.rst

```diff
@@ -5,47 +5,37 @@
    :class: compatibility-large
```

Comment:
also looks right for Rust. I will note that our upcoming 2.2 release will add support for both 5.1 and 5.2 excluding one feature: supporting $merge and $out on secondaries.
I'll also note that 2.2 will add support for change streams so we will be able to remove that footnote then